draft/spec/index.html

```diff
@@ -437,9 +437,9 @@ <h2>Object Structure</h2>
         <h2>Object Conformance Declaration</h2>
         <p>
             The OCFL specification version declaration <span id="E002">MUST</span> be formatted according to the
-            [[!NAMASTE]] specification.
-            It <span id="E003">MUST</span> be a file in the base directory of the OCFL Object Root giving the OCFL
-            version in the filename. The filename <span id="E004">MUST</span> conform to the pattern
+            [[!NAMASTE]] specification. There <span id="E003">MUST</span> be exactly one version declaration file in
+            the base directory of the OCFL Object Root giving the OCFL version in the filename.
```

Comment:
Unrelated to the substance of this PR, should this be `<a>OCFL Object Root</a>` here?

Reply:
That would be consistent with the OCFL Storage Root sentence. Have added